### PR TITLE
DOCSP-3740: adds landing page for Paris .local

### PR DESCRIPTION
**Pull request by schmalliso**

### CODE REVIEW

**Comment by schmalliso** [comment]
Rendered site as PDF: [MongoDB Paris .local-corrected.pdf](https://github.com/mongodb/docs-conferences/files/2621005/MongoDB.Paris.local-corrected.pdf)

File: build/master/html/paris.html [hunk 1]
```diff
@@ -0,0 +1,1348 @@
...
+<li class="toctree-l5"><a class="reference internal" href="mongodb/actions/collection.findOne/#finding-a-document-matching-a-regular-expression">Finding a Document Matching a Regular Expression</a></li>
+</ul>
+</li>
+</ul>
+</li>
+<li class="toctree-l3"><a class="reference internal" href="mongodb/actions/collection.count/">collection.count()</a><ul>
+<li class="toctree-l4"><a class="reference internal" href="mongodb/actions/collection.count/#definition">Definition</a></li>
+<li class="toctree-l4"><a class="reference internal" href="mongodb/actions/collection.count/#examples">Examples</a><ul>
+<li class="toctree-l5"><a class="reference internal" href="mongodb/actions/collection.count/#counting-all-documents-in-a-collection">Counting All Documents in a Collection</a></li>
+<li class="toctree-l5"><a class="reference internal" href="mongodb/actions/collection.count/#co
```
Comment (dcoupal) [inline build/master/html/paris.html]: 1) quote "Serverless", or put in italic?
2) qui est compatible avec divers architectures => compatible avec plusieurs architectures.
     OR ... avec des architectures diverses.
Note that architecture is "feminin".

Comment (dcoupal) [inline build/master/html/paris.html]: 1)  un connection => une connection
2)

Comment (dcoupal) [inline build/master/html/paris.html]: installer et démarrer MongoDB et faire vos premiers requêtes
=>
1) I would remove the first "et" and replace it with a comma. Usually only the last association gets the "et"
2) premiers => premières

Comment (dcoupal) [inline build/master/html/paris.html]: 1) le gestion => la gestion

Comment (dcoupal) [inline build/master/html/paris.html]: may be:   monitoring => surveillance
but the Frenchs have a lot of those english words.
Interesting enough, Québécois do too, however often different words.

Comment (dcoupal) [inline build/master/html/paris.html]: => La documentation officielle de MongoDB

Comment (dcoupal) [inline build/master/html/paris.html]: => Créer une application

**Review by dcoupal — CHANGES_REQUESTED** [review]
Overall, very good job!

I am not sure if I hit everything.
Mostly minor gender mistakes.

File: build/master/html/paris.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (schmalliso) [reply build/master/html/paris.html]: Done!

[the remark this answers is not in the document]
Comment (schmalliso) [reply build/master/html/paris.html]: Beaucoup mieux - thanks!

Comment (schmalliso) [reply build/master/html/paris.html]: Done and done. Merci!

Comment (schmalliso) [reply build/master/html/paris.html]: Dangit, I fixed that in the source but didn't update my PR 😸

Comment (schmalliso) [reply build/master/html/paris.html]: Done

Comment (schmalliso) [reply build/master/html/paris.html]: Dangit.

Comment (schmalliso) [reply build/master/html/paris.html]: So convenient. I've changed to surveillance, though 'monitoring' seems relatively popular on french informatique websites as well. 🤷‍♀️

Comment (schmalliso) [reply build/master/html/paris.html]: Done.

File: build/master/html/paris.html [hunk 2]
```diff
@@ -0,0 +1,1348 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  <meta name="robots" content="index" />
+
+  <meta name="release" content=""/>
+  <meta name="version" content="master"/>
+  <meta name="DC.Source" content="https://github.com/mongodb/10gen/baas-docs/blob/master/source/index.txt"/>
+  <meta property="og:image" content="http://s3.amazonaws.com/info-mongodb-com/_com_assets/cms/mongodb-for-giant-ideas-bbab5c3cf8.png">
+  <meta property="og:image:secure_url" content="https://webassets.mongodb.com/_com_assets/cms/mongodb-for-giant-ideas-bbab5c3cf8.png"><meta content="MongoDB Paris .local : La documentation de MongoDB offre des parcours d'apprentissage pour les administrateurs, les développeurs, et les analystes." name="description" />
+<meta content="MongoDB Paris .local : La documentation de MongoDB offre des parcours d'apprentissage pour les administrateurs, les développeurs, et les analystes." name="keywords" />
```
[the remark this answers is not in the document]
Comment (schmalliso) [reply build/master/html/paris.html]: yay

**Review by schmalliso — COMMENTED** [review]
Updated! And here's the new PDF (sorry again about that…).
[MongoDB Paris .local copy-review.pdf](https://github.com/mongodb/docs-conferences/files/2624989/MongoDB.Paris.local.copy-review.pdf)

**Pull request merged**